pywbem_mock/_namespaceprovider.py

```diff
@@ -36,8 +36,8 @@
 from pywbem._nocasedict import NocaseDict
 from pywbem._utils import _format
 
-# TODO: import from pywbem_mock because it fails probably because
-#       of a circular import
+# NOTE: Should import from pywbem_mock since InstanceWriteProvider is in __all__
+#       but that fails probably because of a circular import.
```

Comment:
There are many cases where we import directly from the sub-module with a relative import in order to avoid circular imports. I don't think that this situation here is any different. Can we simply remove the comment?

Reply:
Agreed.